metrics/tarantool/memtx.lua

```diff
@@ -0,0 +1,128 @@
+local utils = require('metrics.utils')
```

Comment:
I think it would be nice to have at least a simple test so things like https://github.com/tarantool/metrics/issues/399 won't happen again

Reply:
add test